### PR TITLE
Template checks

### PR DESCRIPTION
**Pull request by RumovZ**
I chose to only skip the new template checks for importing apkgs, since the user can't do anything about it in this case. On the forum, someone was complaining about not being able to import a csv due to a broken template. But they can just fix the notetype they're importing into, so I don't see any reason to forgive errors in this case.
Let's hope that enforcing the strict rules when editing or exporting is enough to decrease the number of broken templates in circulation!

Can we revert 9c354f5e6b5c59521e36ec9ad3d7efb72da45a99 now? Which import routine is the notetaking tool you mention there targeting?

It turned out, the [deck that was the cause](https://forums.ankiweb.net/t/2-1-45-release-candidate/11362/42) for skipping the checks during import in the first place wasn't broken after all! It's using the `{{Card}}` special field to generate different front templates from the same string literal.
I have no idea why anyone would design a notetype like this, but I have adjusted the uniqueness check to take this edge case into account.

### CODE REVIEW

**Review by dae — COMMENTED** [review]
Thanks for the quick fix Rumo, aside from a minor question of wording, it looks good to me. :-)

**Comment by dae** [comment]
> It turned out, the deck that was the cause for skipping the checks during import in the first place wasn't broken after all! It's using the {{Card}} special field to generate different front templates from the same string literal. 

🤯 

Thanks for spotting that

File: ftl/core/card-template-rendering.ftl [hunk 1]
```diff
@@ -21,6 +21,9 @@ card-template-rendering-conditional-not-open = Found '{ $found }', but missing '
 # when the user referenced a field that doesn't exist
 # eg, Found '{{Field}}', but there is not field called 'Field'
 card-template-rendering-no-such-field = Found '{ $found }', but there is no field called '{ $field }'
+# when the user referenced a field that doesn't exist in a conditional, excepting card numbers like 'c1'
+# eg, Found '{{#Field}}', but neither is it a card number nor is there a field called 'Field'
+card-template-rendering-no-such-conditional = Found '{ $found }', but neither is it a card number nor is there a field called '{ $key }'
```
[the remark this answers is not in the document]
Comment (RumovZ) [reply ftl/core/card-template-rendering.ftl:26]: Sure. In that case we can probably just use the existing string directly above.

**Comment by RumovZ** [comment]
Only this question remains. 🙂 

> Can we revert 9c354f5 now? Which import routine is the notetaking tool you mention there targeting?

**Comment by dae** [comment]
Sorry, meant to answer that. It's a notetaking app called MarginNote that is quite popular in China. If we restore the check, users will be able to import the collections thanks to the changes in this PR, but it will mean they won't be able to make any changes to the notetype without first converting it to a Cloze notetype, which will confuse the users, and probably generate support requests :-(

**Comment by RumovZ** [comment]
Oh, that's too bad.

One thing I forgot to mention: 4d7fcf5 allows for all special fields to be used in conditionals. That doesn't make much sense for anything but 'Tags', but since it doesn't do much harm, either, I didn't want to complicate the code.

**Pull request merged**